gui/widgets/grid.cpp

```diff
@@ -175,26 +185,45 @@ void GridItemWidget::handleMouseMoved(int x, int y, int button) {
 	}
 }
 
+void GridWidget::toggleGroup(int groupID) {
+	// Shrink group if it is expanded
+	if (_groupExpanded[groupID]) {
+		_groupExpanded[groupID] = false;
```

Comment:
_groupExpanded[groupID] = !_groupExpanded[groupID]

Reply:
Fixed in 32eafca.